### ISSUE TITLE
Add support for pt (point) unit

### ISSUE THREAD

**Issue by omishah** [daohoangson/flutter_widget_from_html#264]
Currently, `pt (point) unit` is not supported/rendered by the `HtmlWidget()`.

For example, try the below raw HTML content and you will notice that font size will be normal instead of 36pt.

```
var _html = '''<p>Hello <span style="font-size: 36pt;">user</span>''';
HtmlWidget(_html);
```

For the time being, I am replacing all the `pt` with `px` on the go by using the `.replaceAll()` method on the HTML string:

`var _html = '''<p>Hello <span style="font-size: 36pt;">user</span>'''.replaceAll(new RegExp(r'pt'), 'px');`

**Comment by daohoangson** [comment]
The PR adding support for `pt` has been merged, it'll be released in the next 0.5 version.

**Comment by omishah** [comment]
> `v0.5.0-rc.2020081901` has been released with support for `pt` unit FYI.

Thank you.
